Carthage/Checkouts/stackviewcontroller/Makefile

```diff
@@ -1,16 +1,17 @@
 SDK="iphonesimulator"
 DESTINATION="platform=iOS Simulator,name=iPhone 6"
 PROJECT="StackViewController"
+SCHEME="StackViewController"
```

Comment:
Did you mean to make this change on `StackViewController` repo instead?

Reply:
Carthage did that...